### ISSUE TITLE
[Info] - Utilities are detected as malicious on virustotal and Google Drive

### ISSUE THREAD

**Issue by 1alessandro1** [acidanthera/bugtracker#1347]
Just to inform the end user:

These are the scans from virustotal.com

[disklabel.exe](https://www.virustotal.com/gui/file/4f997a1fe502fbe0f9076db9a02abadc0907fb95011201fb5b56f16a2375c000/detection)

![image](https://user-images.githubusercontent.com/46293832/100865432-9a65c900-3497-11eb-8f6b-400290c62716.png)

[icnspack.exe](https://www.virustotal.com/gui/file/4d2b326ff1b1eec5f0ab5c15c9e50eb4c250e31c0e09fa56d9ac7106f15d88b0/detection)

![image](https://user-images.githubusercontent.com/46293832/100865463-a6518b00-3497-11eb-905f-5550b2cadbd0.png)

[macserial.exe](https://www.virustotal.com/gui/file/9231a99f91c677d7694873f9d98517362cc61febe9acc7598af882052d71ddbf/detection)

![image](https://user-images.githubusercontent.com/46293832/100865488-b0738980-3497-11eb-89a4-8a2a84b968c0.png)

[ocvalidate.exe](https://www.virustotal.com/gui/file/f4ee11f72299e02137b06fd9b20f3e6157c01847fcca0de15d6ec59a7bab8892/detection)

![image](https://user-images.githubusercontent.com/46293832/100865529-bcf7e200-3497-11eb-8278-1bb6988ed97f.png)

These are alerts from Google (Drive)

![2020-12-02 12 09 14](https://user-images.githubusercontent.com/46293832/100865151-435ff400-3497-11eb-8022-e16d764cb339.jpg)
![2020-12-02 12 09 26](https://user-images.githubusercontent.com/46293832/100865154-44912100-3497-11eb-9c87-eb0ef9f863ab.jpg)
![2020-12-02 12 09 31](https://user-images.githubusercontent.com/46293832/100865155-4529b780-3497-11eb-8c41-fb8e89295735.jpg)
![2020-12-02 12 09 36](https://user-images.githubusercontent.com/46293832/100865157-45c24e00-3497-11eb-81fc-ae417ba797f1.jpg)

**Comment by vit9696** [comment]
Hello,

The results come from basically unknown products from unknown companies:

- Bkav
- Ikarus
- Cylance
- BitDefenderTheta
- VBA32

To be frank, the only name I heard is Ikarus, and my association is with a company building buses years ago. Furthermore, they detect it like `Unsafe`, i.e. not malware, but I do not know what it is, let me just call it something. Sorry, but not only these products are trash, but their results are even worse. They show off their incompetence of being unable to analyse any file be it legit or not.

We know that our files definitely do not contain any malicious code (and you can ensure this by checking the sources), so the only concern is whether or not we make any real-world products unhappy. As long as more or less adequate companies like industry leaders — Kaspersky or Symantec — do not moan, I believe we do everything right. If you want to solve the matter with the companies mentioned, try contacting them about false positives and request them to fix each. In general this does not take long to fix it up. If they provide details on what they believe makes their software to report false positives, perhaps we can find an easier way to workaround it. It is usually hard to quickly find where to report such stuff, here is a [page from google](https://safezone.cc/threads/kuda-soobschit-o-lozhnom-srabatyvanii-antivirusa.23501/) containing a list of e-mails.

As for Google Drive, this situation is more surprising, but in general I see the "Google cannot check this file for viruses" message, so most likely any software they have for malware checking is just broken. Feel free to contact them as well. Perhaps they can provide more data.

All in all, our position is "do not care", since it is companies' job to write working antimalware software and we are not paid to help them. Between choosing not shipping Windows binaries at all and shipping Windows binaries that are detected by weird companies, I would opt for the latter, because we value users that install macOS from Windows and try to give them at least some level of support. My own experience with VirusTotal is that even a >50% detect is often not a confirmed piece of malware, so it really is unhelpful.

**Comment by 1alessandro1** [comment]
Thanks for replying, I knew those were just false positives from basically unknown companies - and also, I trust the code since it's open source, everyone can have a look

It was this morning that I was backing up some folders to Google drive and between them there was OpenCore's release zip file, so I narrowed down the search, and those .exes are the only ones which constitute a complaint from Google, and the user which is unaware of what's under the hood may panic for something innocuous

> so most likely any software they have for malware checking is just broken

This is so true ideed.

TL;DR

My intent was more towards "history tracking" to say, "we already knew that those scans are booked and pretty unuseful" 

Of course I trust the code, I wouldn't be here otherwise using OpenCore as my daily (efi) driver (haha)

 Wish you the best and keep up the good work.
